### DISCUSSION TITLE
Option to display max value without having to create customize MultiSelect component

### DISCUSSION THREAD

**Discussion by trananhvan1102vn** [mantinedev/mantine#5875] [Feature requests]
Hello, I had trouble with the MultiSelect. It does not have the option to limit the number of values that can be displayed since it would break the layout if there are too many values or the label is too long.

https://d13dpwul2c36wz.cloudfront.net/image_1709715143207.png

I can find that you guys already had the solution for this
https://mantine.dev/combobox/?e=MaxDisplayedItems

But following this solution will cost to create a customized MultiSelect component and have to implement all handles of it which will make the component can not be updated when changing version.

I have used the solution above to implement into MultiSelect component. I can create a pull request if needed. But still hope you guys can add this feature to avoid risks above

**Comment by rtivital** [comment]
It is not planned to add this feature to MultiSelect component. If you need it build a custom component with Combobox

**Reply by trananhvan1102vn** [reply]
Can I ask if there is any reason behind it? I had created customize MultiSelect with this feature and it works just fine

**Reply by rtivital** [reply]
https://mantine.dev/core/multi-select/#combobox

**Comment by marcus-maro** [comment]
Giving a +1 for this request. I find myself almost always wishing the `MultiSelect` component had a built in `maxDisplayed` prop!
